src/containers/Dashboard.vue

```diff
@@ -151,7 +151,11 @@ export default Vue.extend({
     resizeEventHandler() {
       this.isMobile = window.innerWidth <= 440;
       this.isTablet = window.innerWidth <= 878;
-      this.maxBottomBarTabs = window.innerWidth <= 1347 ? 2 : 4;
+      if (window.innerWidth <= 1347) {
+        this.maxBottomBarTabs = window.innerWidth <= 480 ? 1 : 2;
```

Comment:
Can we define these magic numbers as constants somewhere? I know that most of these values refer to the breakpoint variables:
https://github.com/cornell-dti/course-plan/blob/875d2d17ce41b1e24604128017adfd9a7bf814b4/src/assets/scss/_variables.scss#L37-L40